src/UnwindRustSgx.h

```diff
@@ -0,0 +1,107 @@
+#if !defined(UNWIND_RUST_SGX_H)
+#define UNWIND_RUST_SGX_H
+
+#ifdef RUST_SGX
+
+#define _GNU_SOURCE
+#include <link.h>
+#include <stdarg.h>
+#include <string.h>
+#include <stdint.h>
+
+
+#define stderr NULL
```

Comment:
Maybe use a different sentinel value to avoid mistakes when calling `fprintf` etc.